usaspending_api/awards/models.py

```diff
@@ -137,11 +137,11 @@ def get_queryset(self):
         and the award will no longer match these criteria
         '''
         q_kwargs = {
-            "type": "U",
-            "total_obligation__isnull": True,
+            "latest_transaction__isnull": True,
```

Comment:
It's on line 186 in models.py under Awards. "Empty" summaries never have one, so this check can replace the others.